### PR TITLE
(FM-8216) Switch testing to use Litmus

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

**Comment by MikaelSmith** [comment]
Seems to work. Of course, all the acceptance tests are pending.

File: .fixtures.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+fixtures:
+  repositories:
+    facts: 'git://github.com/puppetlabs/puppetlabs-facts.git'
+    puppet_agent: 'git://github.com/puppetlabs/puppetlabs-puppet_agent.git'
+    provision: 'git://github.com/puppetlabs/provision.git'
```
Comment (ThoughtCrhyme) [inline .fixtures.yml]: no newline at eof

File: .travis.yml [hunk 2]
```diff
@@ -60,4 +121,4 @@ branches:
     - /^v\d/
     - release
 notifications:
-  email: false
+  email: false
```
Comment (ThoughtCrhyme) [inline .travis.yml]: eof nl

File: distelli-manifest.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    - bundle exec rake litmus:acceptance:parallel
+  AfterBuildSuccess:
+    - source /opt/rh/rh-ruby25/enable
+    - bundle exec rake litmus:tear_down
+  AfterBuildFailure:
+    - source /opt/rh/rh-ruby25/enable
+    - bundle exec rake litmus:tear_down
+  CommitData:
+    - RepoType: Git
+    - RepoPath: .
```
Comment (ThoughtCrhyme) [inline distelli-manifest.yml]: eol nl

**Comment by eimlav** [comment]
@ThoughtCrhyme updated with suggested changes.

**Pull request merged**